examples/demos/elation/Elation.jsx

```diff
@@ -59,6 +59,16 @@ export default class Elation extends React.Component {
     alert(`Selected appointment with ${event._patientName} for ${name} starting at ${new Date(event._apptTime).toLocaleString()}`);
   }
 
+  onAvailabilityClick = (availability /*, e*/) => {
+    const name = getPhysicianName(availability.providerId);
+    // eslint-disable-next-line no-undef
```

Comment:
No-op:

An easy way to get rid of this linter error is to use `window.alert` instead of `alert`. Although I don't know the linter config of this repo and you might get a different error instead about not using the native alert function.

Reply:
Yup, good call. Thanks Sidney!